modules/networking/vars.tf

```diff
@@ -8,6 +8,14 @@ variable "stack_name" {
   default     = "COS"
 }
 
+variable "asg_name_backoffice" {
+  description = "ID of the autoscaling group for dc-backoffice."
+}
+
+variable "asg_name_public_services" {
+  description = "ID of the autoscaling group for dc-backoffice."
```

Comment:
Please update to match the content. Most probably it is the `name` and the dc reference.